docs/source/plugins.rst

```diff
@@ -5,30 +5,33 @@
 Plugins
 =======
 
-Plugins that come with the **IBM z/OS core collection** augment Ansible's core
+Plugins that come with the **IBM z/OS core collection** complement Ansible's core
 functionality. Ansible uses a plugin architecture to enable a rich, flexible
 and expandable feature set.
 
 Action
 ------
 
-* ``zos_ping``: Manages the REXX source transferred to the z/OS managed node for
-  `zos_ping`_.
-* ``zos_copy``: Used to `copy data`_ from the controller to the z/OS managed
-  node.
-* ``zos_fetch``: Used to `fetch data`_ from the z/OS managed node to the
-  controller.
-* ``zos_job_submit``: Used to `submit a job`_ from the controller and optionally
-  monitor the job completion.
+Action plugins integrate local processing and local data with module functionality.
+Action plugins are executed by default when an associated module is used; no additional
+user action is required, this documentation is reference only.
 
-.. _normal:
-   https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/action/normal.py
-.. _zos_ping:
-   modules/zos_ping.html
-.. _copy data:
+* `zos_copy`_: Used to copy data from the controller to the z/OS manage node.
```

Comment:
why do some of these, like 'zos_copy'_ have a trailing underscore, while others, like 'zos_ping': not have the underscore?  I may be missing a subtlety of .rst format here.

Reply:
@richp405 , good question; the generated RST from Sphinx is a result of how the source files are documented in this case here:
```
.. _zos_ping:
   modules/zos_ping.html
``` 
is a reusable hyperlink that can be assessed in RST many times over simply by doing this:
```
some text and now a link to `zos_ping`_
```
With regard to `zos_copy`_ that is calling likely something like below:
```
.. _zos_copy:
   modules/zos_copy.html
```